src/config.js

```diff
@@ -445,6 +445,11 @@ const validators = {
       "A comma separated list of contact fields to not ship to the client. Can include 'external_id, cell, and lastName'",
     default: ""
   }),
+  HIDE_10DLC_REGISTRATION_WARNING: bool({
```

Comment:
nit: prefer having affirmative variables as that's a bit easier to use and read.

Reply:
Bad existing practice throwing me off 😆 so also opened #1614